### DISCUSSION TITLE
Is it possible to rebind the gyro controls?

### DISCUSSION THREAD

**Discussion by Sqlava** [r57zone/DualShock4-emulator#79] [Q&A]
I've been looking for something that would allow me to emulate the gyro movement of a controller with my mouse. From the readme it seems like this program does not do that but is that even a possibility? If it's not could I have an explanation as to why it isn't? To me it seems like mouse movement would translate almost directly to gyro. Please and Thank you.

**Comment by r57zone** [comment]
You can't use the mouse, unfortunately, you can use the buttons. In keyboard and mouse mode, the right stick is controlled by the mouse.

**Comment by Sqlava** [comment]
You never really explained why it wasn't possible but it's fine. I figured out how to implement it. Closing this.
